dkms.conf

```diff
@@ -1,7 +1,5 @@
 PACKAGE_NAME="rtl8812AU"
-PACKAGE_VERSION="#MODULE_VERSION#"
+PACKAGE_VERSION="4.3.22"
```

Comment:
This was for the purposes of DKMS, rather than the built module. In Ubuntu's version of DKMS, running `sudo dkms add .` from this directory reads the `dkms.conf` and automatically copies the source code over to `/usr/src/PACKAGE_NAME-PACKAGE_VERSION`. However, it doesn't look like this is standard in all distros, based on Dell's manpage and based on another user for a different module.

I can revert this part, if you wish.